src/components/utils/file.js

```diff
@@ -1,4 +1,4 @@
-import loadImage from 'blueimp-load-image/js/load-image'
+import loadImage from 'blueimp-load-image/js'
```

Comment:
are we not using the previous version? something to change to make the import different?